generators/server/templates/package.json.ejs

```diff
@@ -32,9 +32,9 @@
     "<%= module.name %>": "<%= module.version %>",
   <%_ });
 } _%>
-    "prettier": "<%= this.dependabotPackageJson.devDependencies['prettier'] %>",
-    "prettier-plugin-java": "<%= this.dependabotPackageJson.devDependencies['prettier-plugin-java'] %>",
-    "prettier-plugin-packagejson": "<%= this.dependabotPackageJson.devDependencies['prettier-plugin-packagejson'] %>"
+    "prettier": "<%= this.dependabotPackageJson.devDependencies?.['prettier'] %>",
```

Comment:
Don't get why it is optional

Reply:
This is temporary.
For tests, `dependabotPackageJson` is loaded at the common generator, which are mocked.
So it's missing at tests.

In any case, we should create a `nodeDependencies` object to replace it. Like we have for java:
https://github.com/jhipster/generator-jhipster/blob/1ec2ebbe7fb7fb147654369d9f9164ca6c2e6553/generators/bootstrap-application-server/types.d.ts#L77